test/generator/CMakeLists.txt

```diff
@@ -171,23 +171,23 @@ add_custom_command(OUTPUT "${EC32}.bc"
                    VERBATIM
                    COMMAND clang -O3 -c -m32 -target le32-unknown-nacl-unknown -emit-llvm "$<SHELL_PATH:${EXTERNAL_CPP}>" -o "${EC32}.bc")
 add_custom_command(OUTPUT "${EC32}.cpp"
-                   DEPENDS "${EC32}.bc"
-                   COMMAND binary2cpp external_code_extern_bitcode_32 < "${EC32}.bc" > "${EC32}.cpp")
+                   DEPENDS "${EC32}.bc" binary2cpp
+                   COMMAND $<TARGET_FILE:binary2cpp> external_code_extern_bitcode_32 < "${EC32}.bc" > "${EC32}.cpp")
 
 set(EC64 "external_code_extern_bitcode_64")
 add_custom_command(OUTPUT "${EC64}.bc"
-                   DEPENDS "${EXTERNAL_CPP}"
+                   DEPENDS "${EXTERNAL_CPP}" binary2cpp
```

Comment:
This command doesn't refer to `binary2cpp`

Reply:
Fixed